src/Tools/BuildUtil/BuildUtil/WriteCodeFragmentEx.cs

```diff
@@ -21,7 +21,7 @@ public class WriteCodeFragmentEx : Task
 
         public override bool Execute()
```

Comment:
:question: Do we still need this task or is it available in the versions of MSBuild we're using now?